js/file/shared/File.svelte

```diff
@@ -20,7 +20,7 @@
 	label={label || "File"}
 />
 
-{#if value}
+{#if value && (Array.isArray(value) ? value.length > 0 : false)}
```

Comment:
shouldn't this be:
```suggestion
{#if value && (Array.isArray(value) ? value.length > 0 : true)}
```
because if its not an Array, it is automatically valid?